### ISSUE TITLE
When I use R3nzSkin, the screen will go black when entering the game. What is the reason for this?

### ISSUE THREAD

**Issue by hisit** [R3nzTheCodeGOD/R3nzSkin#1185]
When I deleted the hid.dll file, I was able to enter the game normally again.
I have downloaded other DLLs in 'lssues' and the same situation applies.

**Comment by Pierreluc23** [comment]
The server you play is Tencent server ?

**Comment by hisit** [comment]
> The server you play is Tencent server ? You no mention you play on what server. Also where you have download the hid.dll file ?

Yes, on Tencent server , https://github.com/Xuemantou/R3nzSkin-For-China-Server/releases/tag/3.3.3

**Comment by ez1996** [comment]
I have the same problem

**Comment by backhandpull** [comment]
> > The server you play is Tencent server ? You no mention you play on what server. Also where you have download the hid.dll file ?
> 
> Yes, on Tencent server , https://github.com/Xuemantou/R3nzSkin-For-China-Server/releases/tag/3.3.3

Will this not result in account suspension?  这不会被封号么？

**Comment by Gary-zy** [comment]
Is it useful that the account I banned last time has not been unblocked yet?  我上次封的号还没被解封 这个有用嘛？

**Comment by hisit** [comment]
> 我上次封禁的账号还没解封有用吗？我上次封的号还没被解封 这个有用嘛？
I don't know, when I use it to enter the game, the screen will go black. Deleting it is necessary to enter normally.

**Comment by hisit** [comment]
> > > 你玩的服务器是腾讯服务器吗？你没有提到你在什么服务器上玩。还有你在哪里下载了hid.dll文件？
> > 
> > 
> > 是的，在腾讯服务器上，https://github.com/Xuemantou/R3nzSkin-For-China-Server/releases/tag/3.3.3
> 
> 这不会导致帐户被暂停吗？这不会被封号么？
I don't know, when I use it to enter the game, the screen will go black. Deleting it is necessary to enter normally.

**Comment by Gary-zy** [comment]
> > 我上次封禁的账号还没解封有用吗？我上次封的号还没被解封 这个有用嘛？
> > I don't know, when I use it to enter the game, the screen will go black. Deleting it is necessary to enter normally.

 I want to change the offset myself, but I don't know what the latest offset data is, and I don't know if it supports Tencent server anymore  想自己更改偏移量 但是不知道最新的偏移量数据是多少，并且不知道支不支持马服了
